data/src/main/java/org/oppia/android/data/backends/gae/NetworkModule.kt

```diff
@@ -68,4 +70,11 @@ class NetworkModule {
   fun provideFeedbackReportingService(@OppiaRetrofit retrofit: Retrofit): FeedbackReportingService {
     return retrofit.create(FeedbackReportingService::class.java)
   }
+
+  // Provides the API key to use in authenticating remote messages sent or received. This will be
+  // replaced with a stronger, secret key will in production.
+  @Provides
+  @Singleton
```

Comment:
You should remove this--immutable strings don't need to be scoped since they can be trivially re-retrieved.

Reply:
Done